src/components/Navbar/WNavbar.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { describe, it, expect } from 'vitest'
+
+import { mount } from '@vue/test-utils'
+import WNavbar from './WNavbar.vue'
+
+describe('WNavbar', () => {
+  const wrapper = mount(WNavbar, {
+    props: {
+      navLinks: [
+        {
+          key: 'stake',
+          active: false,
+          to: '/stake',
+          rel: 'stake',
+          locale: 'Stake'
+        },
+        {
+          key: 'buy',
+          active: false,
+          to: '/buy',
+          rel: 'buy',
+          locale: 'Buy'
+        },
+        {
+          key: 'build',
+          active: true,
+          to: '/build',
+          rel: 'build',
+          locale: 'Build'
+        }
+      ],
+      redirectionActive: false
+    }
+  })
+  it('renders properly first link', () => {
+    expect(wrapper.html()).toContain('<a href="#stake"')
+  })
+  it('renders properly second link', () => {
+    expect(wrapper.html()).toContain('<a href="#buy"')
+  })
+  it('renders properly third link', () => {
+    expect(wrapper.html()).toContain('<a href="#build"')
+  })
+  it('renders properly pointer icon when active', () => {
+    expect(wrapper.html())
```

Comment:
I'd replace this test with a snapshot test because it'd test the same thing in a straightforward way